zipline/algorithm.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
         self.algoscript = kwargs.pop('script', None)
 
         self._initialize = None
+        self._analyze = kwargs.pop('analyze', None)
```

Comment:
I don't know of a way to call Travis without requesting a pull, but if you don't want to open a PR against quantopian/zipline master, you could move your changes to a branch and PR against your own master.

Running your branch locally, the issue I'm getting is about metadata missing from the assets db.  I think if you pass in a panel directly to `run`, then it will update the db and create the data source for you.